cli/azd/pkg/auth/errors_test.go

```diff
@@ -100,3 +100,35 @@ func TestReLoginRequired(t *testing.T) {
 		})
 	}
 }
+
+func TestReLoginRequiredDescription(t *testing.T) {
+	tests := []struct {
+		name string
+		resp *AadErrorResponse
+		want string
+	}{
+		{
+			"invalid_grant",
+			&AadErrorResponse{
+				Error:            "invalid_grant",
+				ErrorDescription: "AADSTS50005",
```

Comment:
I will admit that it took a while for me to fully comprehend what is being tested here. I'm wondering if we would consider small improvements here:

- **Naming**. `TestReLoginRequiredError` instead of `TestReLoginRequiredDescription` -- We are testing the `.Error()` behavior; the description is an implementation detail member.
- **Values used in ErrorDescription**. I'm wondering if we would use values that are more descriptive of its' real-world usage. I could imagine that even a value like "description 1" makes it clearer that this is a text field; unlike "AADSTS50005" would suggest an error code field.

Reply:
Thanks for the suggestions, updated